app/templates/player_bar.html

```diff
@@ -22,15 +22,15 @@
             <h2 class="song-name">{{ playerBar.songPlayer.currentSong.title }}</h2>
             <div class="seek-control">
                 <seek-bar value="{{ playerBar.songPlayer.currentTime }}" max="{{ playerBar.songPlayer.currentSong.duration }}" on-change="playerBar.songPlayer.setCurrentTime(value)"></seek-bar>
-                <div class="current-time">2:30</div>
+                <div class="current-time">{{ playerBar.songPlayer.currentTime }}</div>
                 <div class="total-time">{{ playerBar.songPlayer.currentSong.duration }}</div>
             </div>
             <h2 class="artist-song-mobile">{{ playerBar.songPlayer.currentSong.title }}</h2>
             <h3 class="artist-name">{{ playerBar.albumData.artist }}</h3>
         </div>
         <div class="control-group volume">
             <span class="ion-volume-high icon"></span>
-            <seek-bar></seek-bar>
+            <seek-bar value="{{ playerBar.songPlayer.volume }}" max="100" on-change="playerBar.songPlayer.setVolume(value)"></seek-bar>
```

Comment:
When the user moves the thumb, it will call the `on-change` function.  Where does volume get changed?